lib/cmd/common/version.js

```diff
@@ -95,5 +95,25 @@ module.exports = {
     }
     return 'Warning - newer FHC version available (' + this.latest.version + '). To update, run\n' +
       'npm install -g fh-fhc';
+  },
+  //Checking If The Platform Version Can Be Used With This Version Of fhc
+  checkTargetVersion: function(targ, cb) {
+    request(targ + "/" + this.url, function(err, response, body){
+      if(err) return cb(err);
+      var platformDetails;
+
+      try{
+        platformDetails = JSON.parse(body);
+      } catch(e){
+        return cb(new Error("Invalid Platform Data"));
+      }
+
+      //If the platform is less that a certain version, then it can't be used with this version of fhc.
+      if(semver.lt(platformDetails.Release, fhc._minPlatformVersion)){
+        return cb(new Error("The Platform You Are Targeting Is Not Compatible With This Version Of fhc. Please Install A Previous Version"));
```

Comment:
It's be nice if we could call out a specific older point release - maybe parse the semver, then
`"please try installing the previous version: npm install " + semver.major + ".x.x
"